src/test/scala/com/databricks/spark/xml/XmlSuite.scala

```diff
@@ -18,10 +18,8 @@ package com.databricks.spark.xml
 import java.io.File
 import java.nio.charset.UnsupportedCharsetException
 import java.sql.{Date, Timestamp}
-
 import org.scalatest.{BeforeAndAfterAll, FunSuite}
 import org.apache.hadoop.io.compress.GzipCodec
-
```

Comment:
Actually, this ordering is fine. This is described in https://github.com/databricks/scala-style-guide#imports but I could not have some time to add a travis style check.